R/nest.R

```diff
@@ -54,16 +55,19 @@ nest.data.frame <- function(data, ..., .key = "data") {
     return(tibble(!! key_var := list(data)))
   }
 
-  out <- dplyr::select(data, !!! syms(group_vars))
+  if (packageVersion("dplyr") < "0.7.99") {
```

Comment:
Should this be `< "0.8.0"`?

Reply:
Yes, i only recently actually made it 0.8.0 on dplyr.

Reply:
I updated the pr this week, it was one box to tick in the release issue in dplyr